component-samples/http-device-sample/sample.properties

```diff
@@ -0,0 +1,45 @@
+#
```

Comment:
What all changes would need to be done in order for the device component-sample to consume this file?

Reply:
documented at the top of the file.